scope/tag.go

```diff
@@ -24,7 +24,8 @@ func Tag(my, sv *Extent) error {
 	if err != nil {
 		return err
 	}
```

Comment:
This 'if' statement is redundant.  Must have never been caught when it was initially checked in.

Reply:
Oh my bad the code was hidden and I had to expand it too see the rest of the code.  This is totally fine, ignore my previous comment.  The unchanged code expander thing wasn't showing up or I didn't see it.